src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -155,14 +155,15 @@ public void actionPerformed(final ActionEvent e) {
             + "Scrolling the mouse wheel will move the map up and down.<br>" + "<br><b> Zooming Out</b><br>"
             + "Holding ALT while Scrolling the Mouse Wheel will zoom the map in and out.<br>"
             + "Select 'Zoom' from the 'View' menu, and change to the desired level.<br>"
-            + "<br><b> Turn off Art</b><br>"
+            + "<br><b> Turn off Map Artwork</b><br>"
```

Comment:
Without context of what "Art" means, this can be a bit confusing. Map Artwork is a little bit more specific/descriptive.